src/include/OSL/rendererservices.h

```diff
@@ -200,17 +200,14 @@ class OSLEXECPUBLIC RendererServices {
 
     /// Given the name of a texture, return an opaque handle that can be
     /// used with texture calls to avoid the name lookups.
-    virtual TextureHandle * get_texture_handle (ustring filename);
+    virtual TextureHandle * get_texture_handle (ustring filename,
```

Comment:
We need to make sure that the commit message is super clear that we are breaking compatibility with RendererServices. Renderers will need to update on their side. As soon as it's merged, we might want to also post to osl-dev just to be sure everybody knows.